packages/web/src/components/gcds-card/stories/gcds-card.stories.tsx

```diff
@@ -123,97 +138,111 @@ const TemplatePlayground = args =>
 <gcds-card
   card-title="${args.cardTitle}"
   ${args.href ? `href="${args.href}"` : null}
-  ${args.type != 'link' ? `type="action"` : null}
-  ${args.titleElement != 'a' ? `title-element="${args.titleElement}"` : null}
-  ${args.tag ? `tag="${args.tag}"` : null}
+  ${args.cardTitleTag != 'a' ? `card-title-tag="${args.cardTitleTag}"` : null}
+  ${args.badge ? `badge="${args.badge}"` : null}
   ${args.description ? `description="${args.description}"` : null}
   ${args.imgSrc ? `img-src="${args.imgSrc}"` : null}
   ${args.imgAlt ? `img-alt="${args.imgAlt}"` : null}
+  ${args.lang != 'en' ? `lang="${args.lang}"` : null}
 >
-  ${args.footer ? `<div slot="footer">${args.footer}</div>` : null}
+  ${args.default ? `${args.default}` : null}
 </gcds-card>
 `.replace(/\s\snull\n/g, '');
 
 export const Default = Template.bind({});
 Default.args = {
   cardTitle: 'Card title link',
   href: '#',
-  type: 'link',
-  titleElement: 'a',
-  tag: 'Tag',
+  cardTitleTag: 'a',
+  badge: 'badge',
   description:
-    'Description or supporting text relating to the headline. Longer text will be truncated with ...',
+    'Description or supporting text relating to the headline.',
   imgSrc: '',
   imgAlt: '',
-  footer: '',
+  default: '',
+  lang: 'en',
 };
 
-export const Link = Template.bind({});
-Link.args = {
+export const Description = Template.bind({});
+Description.args = {
   cardTitle: 'Card title link',
   href: '#',
-  type: 'link',
-  titleElement: 'a',
-  tag: '',
+  cardTitleTag: 'a',
+  badge: '',
   description:
-    'Description or supporting text relating to the headline. Longer text will be truncated with ...',
+    'Description or supporting text relating to the headline.',
+  imgSrc: '',
+  imgAlt: '',
+  default: '',
+  lang: 'en',
+};
+
+export const Slot = Template.bind({});
+Slot.args = {
+  cardTitle: 'Card title link',
+  href: '#',
+  cardTitleTag: 'a',
+  badge: '',
+  description: '',
   imgSrc: '',
   imgAlt: '',
-  footer: '',
+  default:
+    'Description or supporting text relating to the headline.',
```

Comment:
Can we use the `<gcds-text/>` component here to try to get the users to use them as well if they are using the slot?